brig/cmd/brig/commands/project_create_test.go

```diff
@@ -8,13 +8,13 @@ import (
 const testProjectSecret = "./testdata/project_secret.json"
 
 func TestInitProject(t *testing.T) {
-	p := newProject()
-	if p.Name != defaultProject.Name {
+	p := newProjectVCS()
```

Comment:
Perhaps we were intending to do so after this PR exited Draft status, but can we add unit tests added around a NoVCS project?  (Checking corresponding defaults, etc.)

Reply:
Yay for more tests.